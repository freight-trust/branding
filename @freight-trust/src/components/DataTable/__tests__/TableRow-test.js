/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import { mount } from 'enzyme';
import { Table, TableBody, TableRow } from '../';

describe('DataTable.TableRow', () => {
  it('should render', () => {
    const wrapper = mount(
      <Table>
        <TableBody>
          <TableRow className="custom-class" />
        </TableBody>
      </Table>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
