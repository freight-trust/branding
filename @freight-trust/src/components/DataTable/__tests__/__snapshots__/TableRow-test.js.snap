// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableRow should render 1`] = `
<Table
  isSortable={false}
  overflowMenuOnHover={true}
>
  <table
    className="bx--data-table bx--data-table--no-border"
  >
    <TableBody
      aria-live="polite"
    >
      <tbody
        aria-live="polite"
      >
        <TableRow
          className="custom-class"
        >
          <tr
            className="custom-class"
          />
        </TableRow>
      </tbody>
    </TableBody>
  </table>
</Table>
`;
